src/network/NetworkController.ts

```diff
@@ -299,12 +296,12 @@ export class NetworkController extends BaseController<
   }
 
   /**
-   * Convenience method to update provider RPC settings
+   * Convenience method to update provider RPC settings.
    *
-   * @param rpcTarget - RPC endpoint URL
-   * @param chainId - Network ID as per EIP-155
-   * @param ticker? - Currency ticker
-   * @param nickname? - Personalized network name
+   * @param rpcTarget - The RPC endpoint URL.
+   * @param chainId - The chain ID as per EIP-155.
+   * @param ticker - The currency ticker.
```

Comment:
If an argument is optional should it have a question mark at the end to match the function signature? Seems like if you look up in the diff there's a spot where the question marks are there, so not sure if we should be consistent about that or not, or whether it matters.

Reply:
I'm not really sure where the question mark syntax came from - that isn't correct JSDoc syntax for optional parameters according to their docs: https://jsdoc.app/tags-param.html

The two documented styles are `{string} [optional]` (JSDoc style) and `{string=} optional` (Closure style). We use the JSDoc style in JavaScript repos, but in TypeScript it's not necessary - the types tell us that the parameter is optional.